supabase/pending/remove_opengraph.sql

```diff
@@ -100,7 +60,32 @@ CREATE OR REPLACE VIEW public.live_specs_ext
     LATERAL view_user_profile(p.user_id) u(user_id, email, full_name, avatar_url);
 COMMENT ON VIEW public.live_specs_ext
     IS 'View of `live_specs` extended with metadata of its last publication';
-
-alter view public.live_specs_ext owner to authenticated;
-
-
+alter view live_specs_ext owner to authenticated;
+
+-- Extended view of user draft specifications.
+create view draft_specs_ext as
```

Comment:
@jgraettinger in response to your [question](https://github.com/estuary/animated-carnival/pull/67#discussion_r973117196) about why I'm recreating `draft_specs_ext` here, it's because we have to recreate `live_specs_ext` to remove the `connector_open_graph` field, and `draft_specs_ext` depends on it. Nothing is changing in `draft_specs_ext` itself.